src/app/pages/CreateHabit/CreateHabit.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import BackButton from '../../components/BackButton/BackButton';
+import Button from '../../components/Button/Button';
+import ColorSelector from '../../components/ColorSelector/ColorSelector';
+import HabitInput from '../../components/HabitInput/HabitInput';
+import styles from './CreateHabit.module.css';
+
+function CreateHabit(): JSX.Element {
+  const [color, selectColor] = useState('red');
+  console.log(color);
```

Comment:
```suggestion
```

Reply:
@lmachens how can I push if the color variable is unused without the console.log?

Reply:
nvm, got it!